packages/core/src/button/_variables.scss

```
@@ -7,9 +7,15 @@ $button-border-width: var(--button-border-width, var(--border-width-base, $borde
 $button-border-radius: var(--button-border-radius, var(--border-radius-sm, $border-radius-md));
 $button-border-radius-max: var(--button-border-radius-max, var(--border-radius-max, $border-radius-max));
 $button-transition-duration: var(--button-transition-duration, var(--animation-duration-fast, $animation-duration-fast));
+$button-active-opacity: var(--button-active-opacity, 0.1);
 
 $button-disabled-opacity: var(--button-disabled-opacity, var(--disabled-opacity, $disabled-opacity));
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)